docker/Dockerfile-build-aggregator.jvm

```diff
@@ -1,42 +1,32 @@
-####
-# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
-###
-
-# Build the project
 FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
 USER root
 COPY . /home/jboss
 WORKDIR /home/jboss
 RUN ./mvnw clean package -DskipTests --projects :checkstyle,:notifications-aggregator
 
-# Build the container
```

Comment:
This comment I would keep as not everybody is yet familiar with multi-stage container builds